### ISSUE TITLE
Line breaks in ACF fields aren't rendered on the frontend.

### ISSUE THREAD

**Issue by louisreingold** [soflyy/oxygen-bugs-and-features#59]
**Describe the bug**
Line breaks in ACF fields aren't rendered on the frontend.

https://www.facebook.com/groups/1626639680763454/permalink/2097127223714695/

**Comment by drank** [comment]
Small correction - the OP in facebook says that BRs are not shown when using the "PHP Function Return Value" option with the ACF's get_field() function but in fact it works without a problem (I tested with an ACF field called 'test' on a page and added it with 'get_field' as function and 'test' as option). Maybe this has already been fixed in 2.2.1.

The actual problem (and I also hit this) is when using the "ACF Custom Field" option to retrieve a text field that contains HTML tags (br, span, etc.). The problem comes from line 573 in "component-framework/includes/acf/oxygen-acf-integration.php":

```
switch( $field['type'] ) {
    case 'text':
    case 'textarea':
    case 'wysiwyg':
    case 'password':
    case 'range':
    case 'oembed':
    case 'color_picker':
    case 'time_picker':
    case 'number':
    case 'email':
        $output = sanitize_textarea_field( $field['value'] ); // <= line 573
        break;
```
That sanitize_textarea_field call actually removes all HTML from the field. Changing that line to `$output = $field['value'];` fixes the problem.
